resources/qml/SidebarHeader.qml

```diff
@@ -243,6 +244,13 @@ Column
             }
         }
 
+        Text  // to take the space of the material info button when the active machine doesn't have multiple extruders
```

Comment:
As explained on the main page of this pull request, the page is about which combinations of materials are viable, not about generic material information.